tests/ignite/distributed/test_auto.py

```diff
@@ -25,14 +25,23 @@ def _test_auto_dataloader(ws, nproc, sampler_name=None, dl_type=DataLoader):
 
     # Test auto_dataloader
     assert idist.get_world_size() == ws
-    dataloader = auto_dataloader(data, batch_size=10, num_workers=2, sampler=sampler, shuffle=sampler is None)
+    NUM_WORKERS = 2
+    dataloader = auto_dataloader(
+        data, batch_size=batch_size, num_workers=NUM_WORKERS, sampler=sampler, shuffle=sampler is None
+    )
 
     assert isinstance(dataloader, dl_type)
     if hasattr(dataloader, "_loader"):
         dataloader = dataloader._loader
+    if ws < batch_size:
+        assert dataloader.batch_size == batch_size // ws
+    else:
+        assert dataloader.batch_size == batch_size
+    if ws < NUM_WORKERS:
```

Comment:
Probably, here comparision does not respect equality case.

Reply:
Done